packages/asset-swapper/test/quote_requestor_test.ts

```diff
@@ -102,9 +114,40 @@ describe('QuoteRequestor', async () => {
                 responseData: unsignedOrder,
                 responseCode: StatusCodes.Success,
             });
+            // A successful response code and good order but for the wrong takerAddress
+            const orderWithNullTaker = testOrderFactory.generateTestSignedOrder({
+                makerAssetData,
+                takerAssetData,
+                expirationTimeSeconds: new BigNumber(makeThreeMinuteExpiryJson()),
+                takerAddress: constants.NULL_ADDRESS,
+                feeRecipientAddress: '0x0000000000000000000000000000000000000002',
+            });
+            mockedRequests.push({
+                endpoint: 'https://425.0.0.1',
+                requestApiKey: apiKey,
+                requestParams: expectedParams,
+                responseData: orderWithNullTaker,
+                responseCode: StatusCodes.Success,
+            });
+
+            // Shouldn't ping an RFQ-T provider when they don't support the requested asset pair.
+            // (see how QuoteRequestor constructor parameters below don't list
+            // any supported asset pairs for this maker.)
+            mockedRequests.push({
+                endpoint: 'https://426.0.0.1',
+                requestApiKey: apiKey,
+                requestParams: expectedParams,
+                responseData: successfulOrder1,
```

Comment:
Should this return a different successful order to ensure it doesn't get included?

Reply:
Good catch, fixed here: https://github.com/0xProject/0x-monorepo/pull/2574/commits/2922ebd09583303f571f9b27e884d4a38f93f456